### DISCUSSION TITLE
Personalyse starter template

### DISCUSSION THREAD

**Discussion by padeville** [vercel/community#2377] [Idea]
In pro/compagny environment
I think it could be a good idea to be able to personalize this starter template 
![image](https://github.com/vercel/community/assets/48120835/5072d7b7-27be-4533-b454-0eb762d9f270)

**Comment by amyegan** [comment]
Hi @padeville! You can customize any of these basic framework starters after your new code repository is created on GitHub, GitLab, or Bitbucket. Or you can click [Browse All Templates](https://vercel.com/templates) at the bottom of that section to find more specialized template options.

**Reply by padeville** [reply]
My request is more about having our company templates and not the vanilla templates 

And I didn't see anywhere how can I add a template in the vercel template ?
